libs/shared/console-shared/src/lib/flow-create-variable/ui/variable-row/variable-row.tsx

```diff
@@ -39,39 +49,77 @@ export function VariableRow(props: VariableRowProps) {
               message: 'Variable name cannot contain spaces.',
             },
           }}
-          render={({ field, fieldState: { error } }) => (
-            <InputTextSmall
-              className="mr-3 flex-1 shrink-0 grow"
-              name={field.name}
-              onChange={field.onChange}
-              value={field.value}
-              error={error?.message}
-              label="Variable"
-              errorMessagePosition="left"
-            />
-          )}
+          render={({ field, fieldState: { error } }) =>
+            watchReadOnly ? (
+              <div className="mr-3 flex items-center gap-2 truncate rounded border border-neutral-200 bg-neutral-100 px-2 text-sm text-neutral-350">
```

Comment:
Missing tooltip for long values

<img width="216" alt="Screenshot 2024-07-30 at 16 33 59" src="https://github.com/user-attachments/assets/4d06d5d3-8633-4d7f-a810-101388402737">